x-pack/test/alerting_api_integration/common/fixtures/plugins/alerts/server/plugin.ts

```diff
@@ -94,21 +96,23 @@ export class FixturePlugin implements Plugin<void, void, FixtureSetupDeps, Fixtu
             read: ['alert'],
           },
           alerting: {
-            read: [
-              'test.always-firing',
-              'test.cumulative-firing',
-              'test.never-firing',
-              'test.failing',
-              'test.authorization',
-              'test.validation',
-              'test.onlyContextVariables',
-              'test.onlyStateVariables',
-              'test.noop',
-              'test.unrestricted-noop',
-              'test.patternFiring',
-              'test.throw',
-              'test.longRunning',
-            ],
+            rule: {
```

Comment:
Would it make sense to add an integration test that grants access to `alerting.alert`? If I'm following along correctly, all of the test fixtures grant `alerting.rule`, but they don't exercise any of the `alerting.alert` paths.

Reply:
@legrego This is a little tricky because we are adding this granularity to support privileges on `rules` and `alerts` but right now the alerting framework is only CRUD-ing rules. We haven't hooked up any of the stack rules (that the alerting framework is responsible for) to actually write out the alerts that this feature privilege might grant access to. Getting the stack rules to write out alerts using the alert data service is something we do plan to do so maybe the functional tests would make more sense in that context when we have actual data to control access to?

@yctercero Are you adding functional tests in your RBAC PR?

Reply:
@ymao1 makes sense, I'm happy to defer this until we have actual data to control access to